src/Command/BaseCommand.php

```diff
@@ -47,7 +47,13 @@
  */
 abstract class BaseCommand extends Command
 {
+    /**
+     * @var InputInterface
+     */
```

Comment:
These annotations (and others) got added back, because phpstan only infers types for private properties that get set in the constructor.

So if they have a default, and dont get set in the constructor we need to add them, or if they are protected/public